kuryr_kubernetes/controller/drivers/lbaasv2.py

```diff
@@ -528,19 +548,29 @@ def _create_loadbalancer(self, loadbalancer):
 
     def _find_loadbalancer(self, loadbalancer):
         lbaas = clients.get_loadbalancer_client()
-        response = lbaas.load_balancers(
-            name=loadbalancer['name'],
-            project_id=loadbalancer['project_id'],
-            vip_address=str(loadbalancer['ip']),
-            vip_subnet_id=loadbalancer['subnet_id'],
-            provider=loadbalancer['provider'])
+        request = {
+                'project_id': loadbalancer['project_id'],
+                'vip_address': str(loadbalancer['ip']),
+                'vip_subnet_id': loadbalancer['subnet_id']}
+        if loadbalancer['name'] != K8S_DEFAULT_SVC_NAME:
+            request['name'] = loadbalancer['name']
```

Comment:
why only adding the name if it is not the default one? perhaps that needs some comment

Reply:
To make sure we continue managing other loadbalancers as before.